src/components/developers/developers.styles.js

```diff
@@ -0,0 +1,99 @@
+import { createUseStyles } from 'react-jss'
+
+const useDevelopersStyles = createUseStyles(theme => ({
+  section: {
+    padding: theme.spacing(2),
+    [theme.breakpoints.sm]: {
+      padding: '96px 72px'
+    }
+  },
+  section1: {
+    paddingTop: theme.spacing(23),
+    backgroundColor: theme.palette.primary.light,
+    display: 'flex',
+    '& $content': {
+      marginRight: 'auto',
+      marginLeft: 0
+    }
+  },
+  section2: {
+    '& $col3': {
+      margin: 0
+    }
+  },
+  section2Text: {
+    paddingTop: theme.spacing(3),
+    paddingBottom: theme.spacing(10)
+  },
+  section3: {
+    backgroundColor: theme.palette.primary.light,
+    textAlign: 'center',
+    '& $col2': {
+      textAlign: 'left'
+    }
+  },
+  box: {
+    background: theme.palette.white,
+    borderRadius: 20,
+    padding: `${theme.spacing(8)}px ${theme.spacing(10)}px ${theme.spacing(8)}px ${theme.spacing(6.5)}px`,
+    fontSize: '23px',
```

Comment:
theme.spacing here? or are you leaving these for later too?

Reply:
Leaving it for later, so I can go trough all.